packages/fxa-settings/src/images/en.ftl

```diff
@@ -0,0 +1,12 @@
+## Images - these are all aria labels used for illustrations
+
+hearts-broken-image-aria-label =
+  .aria-label = A computer and a mobile phone an image of a broken heart on each
+hearts-verified-image-aria-label =
+  .aria-label = A computer and a mobile phone and a tablet with a pulsing heart on each
+signin-recovery-code-image-description =
+  .aria-label = Document that contains hidden text.
+signin-totp-code-image-label =
+  .aria-label = A device with a hidden 6-digit code.
```

Comment:
I also noticed this! The reason for the variation in the ftl ids (and the reason why I didn't update the strings to remove the periods) is because these strings had already been merged in for localization, so i didn't want to change them and trigger them having to be re-translated. If I'm wrong in my understanding of that I'm happy to update!

Reply:
Yes, the ID change would trigger re-translation and for something relatively minor like this probably not worth it.